src/Apps/Artwork/Components/ArtworkDetails/ArtworkDetailsArticles.tsx

```diff
@@ -22,10 +23,11 @@ export const ArtworkDetailsArticles: React.SFC<
       <Col>
         <Box>
           {articles.map((article, index) => {
+            const imageUrl = get(article, p => p.thumbnail_image.resized.url)
```

Comment:
🙌 I love it so much. @damassi getting some good usage out of this `get` function.

Reply:
lol this `get` was a long time coming....